ANGULAR/Task_2/User-Management-App/src/app/active/active-component/active-component.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-active-component',
+  templateUrl: './active-component.component.html',
+  styleUrls: ['./active-component.component.css']
+})
+export class ActiveComponentComponent implements OnInit {
+  route = "active";
+  constructor() { }
```

Comment:
Why are we having blank lifecycle hooks? We should not use any hooks if it not needed